oclc/linked_data.py

```diff
@@ -187,7 +187,9 @@ def lookup(self, identifier_or_uri, processed_uris=set()):
         """Perform an OCLC Open Data lookup for the given identifier."""
         type = None
         identifier = None
-        if isinstance(identifier_or_uri, basestring):
+        if isinstance(identifier_or_uri, bytes):
+            identifier_or_uri = identifier_or_uri.decode("utf8")
+        if isinstance(identifier_or_uri, unicode):
```

Comment:
Here I do want to keep open the possibility that we're looking at a bytestring, because URIs for lookups can come in from outside.